healthcheck-sofa-boot-starter/src/main/java/com/alipay/sofa/healthcheck/core/HealthIndicatorProcessor.java

```diff
@@ -59,6 +57,9 @@ public void init() {
             Assert.notNull(applicationContext, "Application must not be null");
             Map<String, HealthIndicator> beansOfType = applicationContext
                 .getBeansOfType(HealthIndicator.class);
+            if (ClassUtils.isPresent("reactor.core.publisher.Flux", null)) {
```

Comment:
Should it be better to move this magic classname to Constant? And since ReactiveHealthIndicator return Mono , I think it will a litter more clear if we use Mono classname here.

Reply:
@straybirdzls  fix.